### ISSUE TITLE
pedagogical suggestions

### ISSUE THREAD

**Issue by maxim-uvarov** [cablehead/nu-adventure#1]
I believe that in the very first steps of teaching, introducing git cloning and import commands is a bit too much, as those actions are very abstract and don't have any relevant objects in the world to which they can be connected.

https://github.com/cablehead/nu-adventure/blob/952c3206f94d85f568f970fc42c37f71bcf5f406/README.md?plain=1#L60-L61

I would recommend just do 2-3 actions simple moves, that might have an understandable connections to the real world. 

look, go, grab, open, apply - understandable things. I think it would be better just to use them at the beginning. To teach how to press keys, how to read the output. 

And only after girls obtain this skills - to teach them the basic pure nushell abstractions. 

And another thing - I would suggest you just to show them yourself the first mission completions, and just to ask to repeat after you.

**Comment by cablehead** [comment]
>  introducing git cloning and import commands is a bit too much

haha, totally. those steps are intended for the assistant to run. I handed them laptop after the `bat ../intro.md`

I'm trying to decide if I should _really_ lock down the environment: so it's only possible to type commands that are intended to be used. e.g. you could break out of the game instantly with cd ../character.. or if leaving things with raw Nushell available adds to the experience for young learners. - If I do go the "really lock things down route" is building on top off Nushell helpful, vs a basic game parsing loop?

And then I also need to decide: am I conflating goals? Should I narrow focus on either intro to the command line, but perhaps not for people who can barely read, or: focus on introducing computers at a raw level for 7 y/o. Not sure I made sense with the last point? heh.

I appreciate your interest! It's just what I was hoping for posting these notes

**Comment by cablehead** [comment]
> look, go, grab, open, apply - understandable things. I think it would be better just to use them at the beginning. To teach how to press keys, how to read the output.

Yeah, really locking things down, at least a first, seems important. I wonder if it's possible to "strip" out all Nushell's builtin to expose are really limited environment?
